### ISSUE TITLE
configure without openCL

### ISSUE THREAD

**Issue by jfoug** [magnumripper/JohnTheRipper#1589]
```
configure: creating ./fmt_externs.h
In file included from opencl_lm.h:13:0,
                 from opencl_lm_fmt_plug.c:9:
common-opencl.h:22:19: fatal error: CL/cl.h: No such file or directory
compilation terminated.
configure: creating ./fmt_registers.h
In file included from opencl_lm.h:13:0,
                 from opencl_lm_fmt_plug.c:9:
common-opencl.h:22:19: fatal error: CL/cl.h: No such file or directory
compilation terminated.
In file included from opencl_lm.h:13:0,
                 from opencl_lm_fmt_plug.c:9:
common-opencl.h:22:19: fatal error: CL/cl.h: No such file or directory
compilation terminated.
```

**Comment by jfoug** [comment]
Fails on CirleCI also (no opencl there)

**Comment by magnumripper** [comment]
Oops, but at least I fixed it before even reading this. Fixed in c0560a4

**Comment by magnumripper** [comment]
To my excuse I was stuck with early 90's speed of Internet. Just checking whether I *had* mail took a minute, never mind actually fetching them. Pushing took a minute, pulling took a minute. That warning from CircleCI never came through until I got home. You certainly get used to having 100 Mbps very quickly!
